opensrp-reveal/src/test/java/org/smartregister/reveal/presenter/ListTaskPresenterTest.java

```diff
@@ -98,7 +123,7 @@ public void testOnLocationValidatedCallsStartFormWithCorrectArgumentsForMosquito
 
         PowerMockito.doNothing().when(listTaskPresenterSpy, "startForm", any(), any(), anyString());
 
-        Whitebox.setInternalState(listTaskPresenterSpy, "selectedFeatureInterventionType", Constants.Intervention.MOSQUITO_COLLECTION);
+        Whitebox.setInternalState(listTaskPresenterSpy, "selectedFeatureInterventionType", MOSQUITO_COLLECTION);
```

Comment:
Shouldn't be below ` listTaskPresenterSpy.onLocationValidated();`

Reply:
Not sure I understand why it would need to.

That variable needs to be set because it is tested in `onLocationValidated`'s conditional logic.